front/src/views/serials/serials.view.js

```diff
@@ -0,0 +1,22 @@
+
+export default {
+  name: 'ff-serials',
+  computed: {
+    getSerialsList () {
+      return this.$store.getters.getSerialsList;
+    }
+  },
+  created () {
+    this.$store.dispatch('getSerials');
+  },
+  methods: {
+    getImgSrc (imgPath) {
+      return this.$store.state.baseUrl + '/' + imgPath;
+    }
+  },
+  filters: {
+    spaceBetweenElement (titles) {
+      return titles.map(element => ' ' + element.firstName + ' ' + element.lastName).join();
+    }
```

Comment:
doesn't it exist yet in filters folder? if no, think - if it is useful at other parts of project - should it be here on in `@filters`?